### ISSUE TITLE
Apparent discrepancy between definitions and requirements

### ISSUE THREAD

**Issue by hansenp57** [uspki/policies#124]
1.6.1 Definitions states that "Air-Gapped - Certificate Systems or components that are physcially and logically disconnected from the public internet."

6.7 Network security controls states "For the Root CA, the CA SHALL be operated in a High Security Zone and in an offline or air-gapped state from all other networks."

So what is the actual requirement, air-gapped from the public internet or air-gapped from all other networks.

For my part I would prefer that the definition use the more restrictive "all other networks" for both consistency and better adherence to the CA/B Forum Network Security requirements.

**Comment by hansenp57** [comment]
Additionally, since Offline is defined as "An air-gapped Certificate System..." the requirement statement above could be restated as ""For the Root CA, the CA SHALL be operated in a High Security Zone and in an air-gapped or air-gapped state from all other networks."

suggest rewording something like "For the Root CA, the CA SHALL be operated in a High Security Zone and in an offline (powered off, disconnected) or air-gapped (powered on, disconnected) state from all other networks."

**Comment by lachellel** [comment]
+1
